From 9c2b117313b85762b6e9e530596703bbfbe4b8e4 Mon Sep 17 00:00:00 2001
Message-Id: <9c2b117313b85762b6e9e530596703bbfbe4b8e4.1387382496.git.minovotn@redhat.com>
In-Reply-To: <c5386144fbf09f628148101bc674e2421cdd16e3.1387382496.git.minovotn@redhat.com>
References: <c5386144fbf09f628148101bc674e2421cdd16e3.1387382496.git.minovotn@redhat.com>
From: Gerd Hoffmann <kraxel@redhat.com>
Date: Mon, 9 Dec 2013 12:19:21 +0100
Subject: [PATCH 45/46] Revert "usb-tablet: Don't claim wakeup capability for
 USB-2 version"

RH-Author: Gerd Hoffmann <kraxel@redhat.com>
Message-id: <1386591561-21826-3-git-send-email-kraxel@redhat.com>
Patchwork-id: 56031
O-Subject: [RHEL-7 qemu-kvm PATCH 2/2] Revert "usb-tablet: Don't claim wakeup capability for USB-2 version"
Bugzilla: 1039513
RH-Acked-by: Paolo Bonzini <pbonzini@redhat.com>
RH-Acked-by: Michael S. Tsirkin <mst@redhat.com>
RH-Acked-by: Marcel Apfelbaum <marcel.a@redhat.com>

This reverts commit aa1c9e971e80d25b92908dce3dec7c38b49480ea.

Signed-off-by: Gerd Hoffmann <kraxel@redhat.com>
(cherry picked from commit 690af06aebdfd043a6222de96a535dcba2144caf)
---
 hw/usb/dev-hid.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Signed-off-by: Michal Novotny <minovotn@redhat.com>
---
 hw/usb/dev-hid.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hw/usb/dev-hid.c b/hw/usb/dev-hid.c
index 66c6331..6aadaa0 100644
--- a/hw/usb/dev-hid.c
+++ b/hw/usb/dev-hid.c
@@ -236,7 +236,7 @@ static const USBDescDevice desc_device_tablet2 = {
             .bNumInterfaces        = 1,
             .bConfigurationValue   = 1,
             .iConfiguration        = STR_CONFIG_TABLET,
-            .bmAttributes          = 0x80,
+            .bmAttributes          = 0xa0,
             .bMaxPower             = 50,
             .nif = 1,
             .ifs = &desc_iface_tablet2,
-- 
1.7.11.7

